src/main/scala/scalismo/io/ActiveShapeModelIO.scala

```diff
@@ -56,8 +56,10 @@ object ActiveShapeModelIO {
   }
 
   def writeActiveShapeModel(asm: ActiveShapeModel, file: File): Try[Unit] = {
+
+    val pointModel = PointDistributionModel[_3D, TriangleMesh](asm.statisticalModel.gp)
     for {
-      _ <- StatismoIO.writeStatismoMeshModel(asm.statisticalModel, file)
+      _ <- StatismoIO.writeStatismoPointModel(pointModel, file)
```

Comment:
I would call it writeStatismoPDM instead of PointModel

Reply:
fixed